home/whatsapp.py

```diff
@@ -40,6 +54,7 @@ def create_whatsapp_template(name, body, quick_replies=(), image_id=None):
         "language": "en_US",
         "components": components,
     }
+    print(json.dumps(data, indent=4))
```

Comment:
We can probably remove this print?

Reply:
Yup, sorry it snuck back in when I removed the StructBlocks and had to check what that looked like in the request